test/integration/backward_compatible/parallel/InvocationServiceTest.js

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const RC = require('../../RC');
+const TestUtil = require('../../../TestUtil');
+const CompactUtil = require('../../../integration/backward_compatible/parallel/serialization/compact/CompactUtil');
+const { ClientPingCodec } = require('../../../../lib/codec/ClientPingCodec');
+const { SetAddCodec } = require('../../../../lib/codec/SetAddCodec');
+const { Invocation } = require('../../../../lib/invocation/InvocationService');
+const { expect } = require('chai');
+const sinon = require('sinon');
+const Long = require('long');
+
+describe('ClientInvocationServiceImplTest', function () {
+    let cluster;
+    let client;
+    let connection;
+
+    const testFactory = new TestUtil.TestFactory();
+
+    const checkUrgentInvocation_withNoData = (client) => {
+        const clientMessage = ClientPingCodec.encodeRequest();
+        const invocation = new Invocation(client.invocationService, clientMessage);
+        client.invocationService.doInvoke(invocation);
```

Comment:
why not this: 

```suggestion
       await client.invocationService.invokeUrgent(invocation);
```

similar for the other method